prosessering/src/main/java/no/nav/familie/prosessering/domene/TaskLogg.kt

```diff
@@ -32,15 +32,9 @@ data class TaskLogg(
         val melding: String? = null,
 
         @Column(name = "opprettet_tid", nullable = false, updatable = false)
-        private var opprettetTidspunkt: LocalDateTime? = LocalDateTime.now()
+        val opprettetTidspunkt: LocalDateTime? = LocalDateTime.now()
```

Comment:
Trenger vi Elvis?

Reply:
Sansynligvis ikke. Bare jeg som ikke ville endre på for mye.

Reply:
Jeg tenker også vi kan droppe Elvis?

Reply:
Hva er Elvis? 🤷‍♀

Reply:
Det er vel strengt talt ikke elvis her. Det er ? som betegner om feltet kan være null.